### ISSUE TITLE
send() with filename, which contains spaces works wrong in Firefox

### ISSUE THREAD

**Issue by kykint** [mikehaertl/php-tmpfile#14]
If $filename contains spaces (e.g. "Some cool file.doc"), then in Firefox it saves with only first part of name (in the case of example 'Some').
My found quick fix:
WAS:
```
'Content-Disposition: ' . $disposition .
                '; filename=' . $filename .
                "; filename*=UTF-8''" . rawurlencode($filename)
```
BECOME
```
'Content-Disposition: ' . $disposition .
                '; filename=' . rawurlencode($filename) .
                "; filename*=UTF-8''" . rawurlencode($filename)
```

**Comment by kykint** [comment]
Another and better way is to add double quotes according to RFC: https://www.ietf.org/rfc/rfc2231.txt
```
'Content-Disposition: ' . $disposition .
                '; filename="' . rawurlencode($filename) . '"' .
                "; filename*=UTF-8''" . rawurlencode($filename)
```

**Comment by mikehaertl** [comment]
Looks reasonable to me. I just wonder if we really also need the extra `rawurlencode()` or if adding double qoutes wouldn't suffice. The example in [RFC 6266](https://tools.ietf.org/html/rfc6266#section-5) also only uses double quotes:

```
Content-Disposition: attachment;
                          filename="EURO rates";
                          filename*=utf-8''%e2%82%ac%20rates
```

We simply forgot to add them in #7. Does it also work for you if you only add double quotes?

Sidenote: Actually I'm also a little surprised that Firefox still uses the first line without encoding details. I would have expected that any modern browser can understand the second variant.

**Comment by kykint** [comment]
Yes, double quotes work fine in Firefox

**Comment by mikehaertl** [comment]
Great. I just released 1.1.4 containing this fix. Thanks for bringing this up.
